pkg/crds/dashboard/crds.go

```diff
@@ -41,11 +41,17 @@ func FeatureCRD() crd.CRD {
 
 func List(cfg *rest.Config) (_ []crd.CRD, err error) {
 	result := []crd.CRD{
+		// source: https://github.com/rancher/system-upgrade-controller/blob/master/pkg/upgrade/plan/plan.go#L53-L70
```

Comment:
Nit, but can we change this to reference commit ID we're using in `go.mod` (or a tag if it's more suitable)